src/containers/CharacterContainer.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import Character from '../components/Character/Character';
+import Phrase from '../components/Phrase/Phrase';
+import Picker from '../components/Picker/Picker';
+
+
+function CharacterContainer(){
+    const [head, setHead] = useState('alien');
+    const [middle, setMiddle] = useState('hoodie');
+    const [bottom, setBottom] = useState('skirt');
+    const [phrase, setPhrase] = useState(['Hello My Brudda']);
```

Comment:
Most times you'd want to leave this as an empty string at first so only the phrases the user adds show at the bottom of the screen